config/config.json

```diff
@@ -31,7 +31,7 @@
         "emailId":"team@focusbear.io",
         "unlockMachineButtonText":"Unlock the machine",
         "wrapupButtonText":"Let's wrap up for the day",
-        "stayFocusedText":"OK - Stay focused throughout the day!",
+        "stayFocusedText":"Now let's set up an account for you",
```

Comment:
isn't this shown at the end of the morning routine? Don't we need a separate translation key?

Reply:
"stayFocusedText":"Now let's set up an account for you", is added new and needs a translation for ES